### ISSUE TITLE
Table Editable not working properly

### ISSUE THREAD

**Issue by stanley85** [pimcore/pimcore#9138]
# Bug Report
When creating an editable of type "Table" via "pimcore_table" it is possible to rearrange columns inside the rendered table editor via drag and drop in the admin UI.

### Expected behavior
The expected behaviour is that these changes trigger rearrangement of the data array layer and the change is recognized by the UI resulting in an unsaved document.

### Actual behavior
The change is displayed visually but saving the document and reloading afterwards restores the old state. The rendered frontend always shows the old state. Document preview also shows the old state, even when no reload of the editmode is done.

### Steps to reproduce
Add "pimcore_table" to your view.html add some columns and data to differentiate the column order, rearrange the columns via drag and drop.

### Code example
```html.twig
{{ pimcore_table("demo_table", {
    "defaults": {
        "cols": 2,
        "rows": 2,
        "data": [
            ["1", "A"],
            ["2", "B"],
            ["3", "C"]
        ]
    }
}) }}
```

**Comment by brusch** [comment]
I'd just lock the columns for now.

**Comment by brusch** [comment]
Fixed by #9239
